src/Mod/Draft/draftguitools/gui_groups.py

```diff
@@ -34,13 +34,15 @@
 
 import PySide.QtCore as QtCore
 from PySide.QtCore import QT_TRANSLATE_NOOP
+from PySide import QtGui
 
 import FreeCAD as App
 import FreeCADGui as Gui
 import Draft_rc
 import draftutils.utils as utils
 import draftguitools.gui_base as gui_base
-from draftutils.translate import _tr, translate
+from draftutils.translate import _tr
+from DraftTools import translate
```

Comment:
Do not import from `DraftTools`, you can import from `draftutils.translate`; this module is the one providing the `translate` function.
```py
from draftutils.translate import _tr, translate
```